_docker/drupal/drupal-filesystem/web/config/sync/core.extension.yml

```diff
@@ -7,6 +7,7 @@ module:
   assembly: 0
   author: 0
   automated_cron: 0
+  ban: 0
```

Comment:
@robpblake It looks like these changes will also enable the ban module. Is that intentional?

Reply:
@jordanpagewhite It is installed as a dependency of the `login_security` module. Just to confirm that I don't need to explicitly list it in `composer.json` as it should be installed by the `drupal/core` dependency?

cheers,

Rob

Reply:
Ah, okay, that makes sense. And, yes, that is correct that it will be downloaded as a dependency of Drupal core.